src/PatientChecking/PatientChecking/Controllers/AppointmentController.cs

```diff
@@ -87,5 +91,31 @@ public async Task<IActionResult> Detail(int appointmentId)
             };
             return View(appointmentDetailViewModel);
         }
+
+        public IActionResult Update(AppointmentDetailViewModel appointmentDetailViewModel)
+        {
+            var appointment = new Appointment
+            {
+                AppointmentId = appointmentDetailViewModel.AppointmentId,
+                CheckInDate = DateTime.Parse(appointmentDetailViewModel.CheckInDate),
+                MedicalConcerns = appointmentDetailViewModel.MedicalConcerns,
+                PatientId = appointmentDetailViewModel.PatientId,
+                Status = appointmentDetailViewModel.Status
+            };
+            if(appointment.CheckInDate < DateTime.Now)
```

Comment:
Add a empty line above this for readable. Please check other places

Reply:
I have added a empty line for readable